bundle/edu.gemini.p2checker/src/test/scala/edu/gemini/p2checker/rules/RuleSpec.scala

```diff
@@ -26,7 +26,7 @@ abstract class RuleSpec extends Specification {
   def setup[I <: ISPDataObject](instrument: SPComponentType, programId: String = "")(mod: I => Unit): ISPObservation = {
     val f = POTUtil.createFactory(UUID.randomUUID())
     val p = f.createProgram(null, SPProgramID.toProgramID(programId))
-    val o = f.createObservation(p, null) <| p.addObservation
+    val o = f.createObservation(p, Instrument.none, null) <| p.addObservation
     val i = f.createObsComponent(p, instrument, null) <| o.addObsComponent
```

Comment:
> The equivalent line in RuleSpec.scala:30 (the next file down) remains … is this because we have an instrument here but not in RuleSpec?

Here I left the explicit observation component creation for the instrument because the test case needs a references to it in lines 36-38.  Otherwise I could have done

```scala
val o = f.createObservation(p, Instrument.fromComponentType(instrument), null) <| p.addObservation
```

as I did in `NonSiderealTargetRulesSpec` above.  It could be done this way anyway, but then I'd need to fish out the instrument observation component from the observation.

Reply:
Of course, the drawback of separating the instrument node creation from the observation is that any observation-specific initialization (like setting up the Ghost asterism) won't be done.